Apps/VisionOS/VisionOS/AppIntroView.swift

```diff
@@ -0,0 +1,42 @@
+import MarkdownUI
+import SwiftUI
+
+struct SampleAppIntro: View {
+    static let title = ScreenTitleConfig(
+        "Welcome to **Customer.io** *&*",
+        menuTitle: "App Intro",
+        showVisionProLogo: true
+    )
+
+    @EnvironmentObject var viewModel: ViewModel
+    @ObservedObject var state = AppState.shared
+
+    var body: some View {
+        Markdown {
+            """
+            We're excited to have you onboard as we explore the walk through the capabilities of
+            **Customer.io** within the realm of spacial computing generally and VisionPro specifically.
+
+            This sample app is more than just a demonstration—it's a hands-on guide
+            designed to familiarize you with **Customer.io Swift SDK**. You'll learn not only how to integrate the SDK into your VisionPro app but also how
```

Comment:
```suggestion
            designed to familiarize you with **Customer.io iOS SDK**. You'll learn not only how to integrate the SDK into your VisionPro app but also how
```

Because the product is named "iOS" today, I think it's good to keep this consistent to avoid confusion.